README.md

```diff
@@ -37,14 +37,9 @@ the parameters passed to `ic-admin` from `bin/dfx-sns-sale-propose`.
 
 ### CI
 
-A new stock snapshot can be released by pushing a tag:
-
-```
-RELEASE="release-$(date +"%Y-%m-%d")"
-echo "$RELEASE"
-git tag "$RELEASE"
-git push origin "$RELEASE"
-```
+A new stock snapshot can be released by running the
```

Comment:
No change.

I know the previous version said this, but I'm not sure what "stock snapshot" means.

Did the old instructions not work?

Reply:
"stock snapshot" is what we call the snsdemo snapshot in some places.
I guess the thought was that we'd have custom snapshots as well and have one "stock" snapshot which isn't custom.

The old instructions still work but the new instructions are simpler so it didn't seem useful to keep the old instructions.